pom.xml

```diff
@@ -286,12 +286,21 @@
                     <group>JBoss SET/Tools</group>
                     <packager>JBoss SET</packager>
                     <mappings>
+                        <!-- not very elegant, but only way I found to ensure the proper files
+                             attributes were set was to "override" the first mapping with a second
+                             one ... -->
                         <mapping>
                             <directory>/usr/share/${project.name}</directory>
                             <filemode>644</filemode>
-                            <username>jboss</username>
-                            <groupname>jboss</groupname>
-                            <artifact />
+                            <username>root</username>
+                            <groupname>root</groupname>
+                            <artifact/>
+                        </mapping>
+                        <mapping>
+                            <directory>/usr/share/${project.name}</directory>
```

Comment:
As said on the PR, this is sadly the only solution I found to get the maven rpm plugin to implements the proper right. Short to that, we have to either modify the plugin itself (might be more demanding than what it brangs) and or leave it and fall back to have a spec file and build the rpm with rpmbuild (cumbersome and unpractical)